CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ and in case a rollback is needed, before installing a previous version, the migr
 - Fix backwards sync bug where chain is rolled back too far, especially when restarting Nimbus [#5497](https://github.com/hyperledger/besu/pull/5497)
 - Check to ensure storage and transactions are not closed prior to reading/writing [#5527](https://github.com/hyperledger/besu/pull/5527) 
 - Fix the unavailability of account code and storage on GraphQl/Bonsai [#5548](https://github.com/hyperledger/besu/pull/5548)
+- Use the node's configuration to determine if DNS enode URLs are allowed in calls to `admin_addPeer` and `admin_removePeer` [#5584](https://github.com/hyperledger/besu/pull/5584)
```

Comment:
this will need to move to next release 23.4.3

Reply:
I've created a new section in the change-log under the latest commit, and removed this line from the 23.4.2 section.